logos/cardinalstone_registrars/cardinalstone.svg

```diff
@@ -0,0 +1 @@
tA0WtgHg20P6DJzjaod9CbhO0Hb35hqYRm87GGvctBXrcXgWXJIDswGX3DjDhaVezMChJfHxMqU0kiCn6mTj2QzHlhIsECbV96UtxzylzNXuwSXUwoZDIwCYGNxBG2Aqm/C0colHuWz//la0vFweDKQxXeyJCpf6aMgD93O8wxG83eNxdyo5vfPoAXjnpsT9rfTH1Qm7+rLDbsrHuntMmmLw8OtJvQ7VuC5O228AYH0Pce9WexBn1nyQPGs8OHcJ/Q71AKmFqwRiXrgN+56+vp+H9DBpe4GT58YtfpR0jHGW/+o3WL0zvEP3WNEu4BkDozfzNIobHOHz55C+uQYBH+rn24jLgDDw+iVsMCIgNCUOYYz6AZU2F/W2NVStQAAkX8lRk1rYS8QiYhipWmjItU0TYlMtbLaZ3Beqq5QGjfyF5OKShNtcvK5NFTME8ObqzmwFI+uYs8vqTJ3t8xcrF0nxQWV3Q4oRDoc0y60CDeA3Aa4jTBFVMUWew+UkGyFtSSWv7Hvf36+AhCu69/Lv4i5YuCUaRux3yDgjcF6S6CeBq6EhOJrC74c9RJjusX7LwDNCfYgpc4RxajHuyfIX5Rc9UiYJ91CSf50w8Fd7iXu8a5f96N/pvv9QddzWv78FlIUfiJjcIE7PlRghMg5IpIcSjeSDQu0Af514IQNIuJSsyeK2mthoTQnDJJfwFDWIV9ZPuPhZvhlw5dVCeZ+0mRyj+iyDnfuohYlYOHAEgYp8qMZBpCi5RtLF2kkK3KseVxiE/dbP5xoI6+f7oJzHtNuY9DXGbvpTXe7rhoDCbbu/oJyaR1XY2fDMEm3XYVUzfLsO1W5b8L1Im47yKinuCnYN9JXbRkG62sv6wxpXZmdoAO4/oq30spHvyChwCwm00baIEtgBDKPbcTeuZT7f30N5l3B6sMd7tdvlCfPDhcO4ndr7lIvsb3nvMNpt67ULnfoFPP9aOg4NcEnViBNvGlBjaCf2BxeTGH0IlxFRsLnK5nN08FmEhgABbVMW/Fq2mdQ1Ss7QFxxIDUdOP8nlj3wlNpdCNNG+4xJHaABlps9YPzdVQs7h7QI/BNO5/jraOf+tnFp3SLO/7H9ck4eTz4ugf9BJgI6HK/0zqGIhYX1OoQOCOw3QNsI1T5h28ddxH0f8espetK3KLYH5vKtH9RtIbclQFZNc0rasHoBx/b52Tgqcs7tE5/sFF9zeq92yEvySCwII12hNEkBGmw+38L7I99d4+o4XFxxwjosrNzgphivf3Yc1GoSXBRfv6FaQOpTLvihwUSklB34T4EqSgko8qdLl7ENZRkZSkTYyyZVd8pDJT9hcij1Ppb6UIHkJywlGSl3JzOQ4hTqQfSaFVEPJ5Qn4AkIoS71682Po/5dK0KJaqJx3ruWlMa5l8vsJVi+Ba7gAEj9MMLnZ93afj6+6c8vthVro3fOp7WCf1fhpINzxaicBomhzBa2ASDGUWrhPBk4BoMr7U4UOkaa38A3aMov6PI17aoB67CS5/VhFZNzaqYVf1rZNuA+LXRHO8zN264FJvo/QPrzFqY52I6B/yQCDqmPSPuEqjSS2xPWSSKcEwfd/EBOyBNgNyRXnsLjkanL6pm2kSZ2p5GrW19+X3IkwA2WkOiylqlBxCWjlJHJOcgkJ05BOnCFlVb44ZjlV0T9zkQfX+Xzc6kt8BPynOWjtvQfh/40Zhru7H4u1yoGz4A4d+xwS8pcJA0Pv3u+VWzCLz+Ci1uWs/ehkcfbSMsG8rFrY6FsPLjcB7lRVZT179ttgtG9wzgq/8z0d8kXg4mLS3Yzdg9BGrLSCtmFK9J5ah0bLgsqtxn8wjYPN7yT4v7fm/YXffkPmsIBxwQahm6pFCYt3QPMalVCSKBQj6qRapk7nNldaMMgBawmcuZ05uGR9qKSM59nogDwwqLeQSiIOYCmRGHGx+buGZCplPRjXb7SFj1UTYDlQESaEHx54t1Ob7P4Tojz8GJ3lvBiDONn3N06Q4revUd3ZzEkGu1sYuqcxD9wvHewQu+Qety3YhY9phbbYN0h7j/Xgnuf7D/fx+CBc31anZ45Q9ntauBqbMRxU534b2+tXzXu6wuX5XyJ0scxLpCu8NEOaepVye5yw2ETI+01uiQ96PSNocZv372GbfdlLPQOptAnbE/TeBvfwIxBbELrBUDLcl3M6IwBO4Bmu3gdcWuSRgEA7hqtNQQowYi7rFD1uCbsZWZ2kmmQIQnKVnEC5YyVny/lydUXdqoKDc26pGPgoR888nwEXc2oQMDYYA3s+L634Na4W+iNu1HIsJATKH7TbD8TOLZHncMPVn2m3LTN+bgh32MJPpeLeiI/D8Xx45mw44gY1KBV6KhKanf/CTXlwLwmcAztA2W3vavy065hvK3L56xTupqsr3CAIYx33BDp4zKlUdvs8/GzRF7TzxIW24zzWoANphVugHaOMV181AF1Xfott3KizOkDbKA1bJwAWznVZBwZu8oPqLX5Bcg/Sx7h5Ekpe3OgGHRa4zH85OnDQSaGsA8bWE+tyKfz/Hdxbp93elW+EZ3Az2R7QzBKDqmaJqqudl9uMcctIFHHLakk0TbXQpeTVoZKLG7fC5mLETcDFACRVVq9OltyhQSVFDlhSsvYDrYogkEBK7zeWGljnSF4tlJKL1YPOP2WYlBL9I6VdVoIyFVMwwNLOFx0E+R2v3dbSr1ZuPmZqGENfPwzmXa1wEyFT4Z6SuC0bevOO0TiHU+JOShV+ZfNkZbeRpkzBetLQC4eRHF3oK9xEBx0TW/n64P4lCGgs/xXGbec3GdS5eXCOm9Dgpje4t+CBbHy03dYOv299HEo+bT+qYfeqcMAzNW6sil/owQ8ZLobzLX15uOHsq+A/vlP693EDpb1VohucjMYdzfB9nKfbT+NGR67N+B/b479qYwF1PNzHNZC7atxWUFcDyu0qdi7u76Hdd9ds3f8PzKzyGwe32ugAAAAASUVORK5CYII="/></defs></svg>
```

Comment:
Hey, sorry but this is a base 64 encoded png wrapped as an svg. Please replace with a high quality vector version of the logo